Servers/LookupServer/main.cpp

```diff
@@ -1,13 +1,24 @@
 #include "LookupServer.h"
 #include <LibCore/CEventLoop.h>
+#include <stdio.h>
 
 int main(int argc, char** argv)
 {
     (void)argc;
     (void)argv;
 
+    if (pledge("stdio unix inet cpath rpath fattr ", nullptr) > 0) {
```

Comment:
```suggestion
    if (pledge("stdio unix inet cpath rpath fattr", nullptr) < 0) {
```

Reply:
Well that's embarrassing :) Thanks for catching!